packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/react/integrationsteps.md

```diff
@@ -0,0 +1,130 @@
+### 1: The user navigates to the sign-in page
+
+When the user navigates to your app's sign-in page, call `OktaAuth.idx.start()` to start the sign-in flow.
+
+```javascript
+const { oktaAuth } = useOktaAuth();
+const startFlow = useCallback(async () => {
+  newTransaction = await oktaAuth.idx.start();
+}, [oktaAuth, flow, setTransaction]);
+```
+
+The method returns an `IdxTransaction` object containing field metadata that can be used to create a sign-in page dynamically.
+
+```json
+{
+   "status":"PENDING",
+   "nextStep":{
+      "name":"identify",
+      "inputs":[
+         {
+            "name": "username",
+            "label": "Username",
+            "required": true
+         }
+         ...
+      ]
+   },
+}
+```
+
+Display &mdash; either statically or dynamically using `IdxTransaction` &mdash; an input field for the user's username.
+
+<div class="half border">
+
+![Screenshot showing a sign-in page with a username field and a Next button.](/img/pwd-optional/pwd-optional-sign-in-page.png)
+
+</div>
+
+### 2: The user submits their username
+
+When the user submits their account details, create an object with a `username` property and assign it the value entered by the user.
+
+```javascript
+const inputValues = {
+  username: "johndoe@email.com",
+ };
+```
+
+Send this new object to `OktaAuth.idx.proceed()`.
+
+```javascript
+setProcessing(true);
+const newTransaction = await oktaAuth.idx.proceed(inputValues);
+setTransaction(newTransaction);
+```
+
+>**Note**: You can also start the sign-in flow in a single step by passing the username in `OktaAuth.idx.authenticate()`. See [`idx.authenticate`](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate) in the GitHub docs or [Node Express's](/docs/guides/pwd-optional-sign-in-email/nodeexpress/main/#_2-the-user-submits-their-username) version of this guide to learn more.
+
+### 3. Identity Engine requests new email verification
```

Comment:
Can we remove the word "new" from the chevron in step 3 please?  
It reads "OIE is requesting the verification of a new email" rather than "OIE is requesting a new verification via an old email" which is what is happening as this is a sign-in flow rather than a sign-in flow or a change-your-password flow. Perhaps

3. Identity Engine requests sends user email challenge

Reply:
done.